OWNERS

```diff
@@ -8,7 +8,6 @@ approvers:
   - celebdor
   - chancez
```

Comment:
Chance too, we need to clean this up broadly and make DPP responsible for scrubbing OWNERS of the departed.